src/pages/receipts/csvFileReceipt.js

```diff
@@ -145,6 +146,18 @@ const updateInTransitMapping = (shippedBoxes) => {
   return holdProcessedResult
 }
 
+const materialTypeMapping = (specimenId) => {
+  return (specimenId.split(' ')[1] === '0001' || specimenId.split(' ')[1] === '0002' || specimenId.split(' ')[1] === '0011' || specimenId.split(' ')[1] === '0012' || specimenId.split(' ')[1] === '0021') ?
```

Comment:
- should be `const specId = specimenId.split(' ')[1]; and check null before using split
- think more about using switch/case instead if/else. Performance can be improved